java-security-it/src/test/java/com/sap/cloud/security/test/integration/IasIntegrationTest.java

```diff
@@ -0,0 +1,39 @@
+package com.sap.cloud.security.test.integration;
+
+import com.sap.cloud.security.config.OAuth2ServiceConfiguration;
+import com.sap.cloud.security.config.Service;
+import com.sap.cloud.security.config.cf.VcapServicesParser;
+import com.sap.cloud.security.test.SecurityTestRule;
+import com.sap.cloud.security.token.*;
+import com.sap.cloud.security.token.validation.CombiningValidator;
+import com.sap.cloud.security.token.validation.ValidationResult;
+import com.sap.cloud.security.token.validation.validators.JwtValidatorBuilder;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IasIntegrationTest {
+
+	@ClassRule
+	public static SecurityTestRule rule = SecurityTestRule.getInstance(Service.IAS)
+			.setKeys("/publicKey.txt", "/privateKey.txt");
```

Comment:
No. By default generated keys would be used which also works.

Reply:
Then, I would remove it... but preferably with a IAS specific key... they looks a little bit different than the xsuaa keys... isn't it?

Reply:
IAS tokens are signed differently than XSUAA tokens?

Reply:
No. But the token_keys endpoint provides the token differently. This can be managed with different key-ids....
XSUAA:
https://green-1.authentication.sap.hana.ondemand.com/token_keys
IAS:
https://xs2security.accounts400.ondemand.com/oauth2/certs

So, finally i would like to have both of them in preferably one file, e.g. token_keys.txt with two different entries one as default for ias and another as default for xsuaa...